### ISSUE TITLE
Using in Browser

### ISSUE THREAD

**Issue by Ratstail91** [sethen/markdown-include#15]
Hi, I was really hoping to use this in the browser. Is there any way it can be adapted for that? I'm already using several other markdown plugins client-side, so I hoped this would simply work.

**Comment by sethen** [comment]
Hello,

Can you give me an example of how you would like to see this used in the browser?  I am having a tough time coming up with a use case (mainly because I have never really thought about it).

**Comment by Ratstail91** [comment]
Lets say I have the following markdown file:

    #include 'header.md'
    #include 'games.md'
    #include 'footer.md'

This file is fetched and parsed client-side, using markdown-it. Each time it hit an #include, a markdown-it plugin would make an async request for that file, and inserts it there before it finishes. This way I could have the contents of header.md and footer.md on several different pages.

I know this isn't designed for that, I just originally misinterpreted it that way. It would be cool if this could be adapted like that, though.

**Comment by sethen** [comment]
Yes, that's an interesting use case.  Currently this is only for server side but it could be adapted at some point for client side.  It would just need to be abstracted away a little bit to work as such.  I will leave this open.
